helm-chart/templates/nginx-mesh-api.yaml

```diff
@@ -387,10 +387,6 @@ runAsUser:
   type: MustRunAsNonRoot
 fsGroup:
   type: MustRunAs
-volumes:
-- configMap
-- csi
-- secret
```

Comment:
There's another SCC at the bottom of this file that likely needs attention too, unless you already checked it out.

Reply:
I actually can't remember what that last SCC is for...it's bound to the system authenticated group

Reply:
I was able to remove most of the volumes from that SCC and still deploy. The remaining are required for the API pod to come up.

Reply:
Cool. I'm not sure why we need the two SCCs or how they differ...worth looking into another time probably.